svm/tests/integration_test.rs

```diff
@@ -2666,14 +2666,18 @@ fn svm_metrics_accumulation() {
             &env.processing_config,
         );
 
-        assert_ne!(
-            result
-                .execute_timings
-                .details
-                .create_executor_jit_compile_us
-                .0,
-            0
-        );
+        // jit compilation only happens on non-windows && x86_64
+        #[cfg(all(not(target_os = "windows"), target_arch = "x86_64"))]
```

Comment:
I linked in the description. This is copied to be consistent with when we jit compile [here](https://github.com/anza-xyz/agave/blob/master/program-runtime/src/loaded_programs.rs#L397)